src/sentry/event_manager.py

```diff
@@ -142,7 +143,23 @@ def has_pending_commit_resolution(group):
     if recent_group_link is None:
         return False
 
-    return not ReleaseCommit.objects.filter(commit__id=recent_group_link.linked_id).exists()
+    # commit has been released and is not in pending commit state
+    if ReleaseCommit.objects.filter(commit__id=recent_group_link.linked_id).exists():
```

Comment:
Does it make sense to refactor `recent_group_link` to be named `latest_issue_commit_resolution`?